apps/system/js/statusbar.js

```diff
@@ -19,43 +19,46 @@
 'use strict';
 
 (function(exports) {
+  const AVG_ICON_SIZE = 22;
```

Comment:
The usual size is 16, but the time, batery and connections (which are always displayed) size is almost 30;